src/Makefile

```diff
@@ -383,6 +383,21 @@ else
 	MAYBE_UNINSTALL_REDIS_SYMLINK=
 endif
 
+# Determine whether to use fast_float as a library or not. 
+BUILD_FAST_FLOAT:=yes
```

Comment:
You can't disable fast_float in the current implementation, since it will fail to build. I think we should probably provide an alternative implementation for fast_float when it's unavailable to maximize compatibility. Either that, or we just require all this stuff.

You probably should also do something like add a function like `valkey_strtod` in util.c that can be called more widely that determines which one to build at compile time.

Reply:
I think the concern is about if C++ is available. I assume it's everyone, but people have complained about this type of incompatibility in the past.

Reply:
OK, then it's safer to allow building without fast_float and then `valkey_strtod` can provide a callback using the old implementation.

Regarding the make variable prefix, some use `BUILD` (`BUILD_TLS`) and some use `USE` (`USE_SYSTEMD`, `USE_JEMALLOC`, ...). It seems that `USE` is more common actually.

Reply:
I made fast_float optional and off by default.